src/bennu/executables/bennu-simulink-provider/main.cpp

```diff
@@ -333,6 +336,7 @@ int main(int argc, char** argv)
         ("debug", po::bool_switch(&debug), "print debugging information")
         ("server-endpoint", po::value<std::string>()->default_value("tcp://127.0.0.1:5555"), "server listening endpoint")
         ("publish-endpoint", po::value<std::string>()->default_value("udp://239.0.0.1:40000"), "publishing endpoint");
+        ("publish-rate", po::value<double>()->default_value(0.1), "rate at which updates published from provider to simulation");
```

Comment:
grammar, change to `rate at which updates are published from the provider to the simulation`